internal/commands/ostest/routing.go

```diff
@@ -0,0 +1,49 @@
+package ostest
+
+import (
+	"github.com/snyk/go-application-framework/pkg/configuration"
+
+	"github.com/snyk/cli-extension-os-flows/internal/flags"
+)
+
+// Flow is the type of all the command's flows.
+type Flow string
+
+// The list of all available flows.
+const (
+	LegacyFlow               Flow = "legacy"
+	SBOMReachabilityFlow     Flow = "sbom-reachability"
+	DepgraphReachabilityFlow Flow = "depgraph-reachability"
+	DepgraphFlow             Flow = "depgraph"
+)
+
+// RouteToFlow will determine which flow to route the command to, based on the provided configuration.
+func RouteToFlow(cfg configuration.Configuration) Flow {
+	ffRiskScore := cfg.GetBool(FeatureFlagRiskScore)
+	ffRiskScoreInCLI := cfg.GetBool(FeatureFlagRiskScoreInCLI)
+	riskScoreFFsEnabled := ffRiskScore && ffRiskScoreInCLI
+	riskScoreThreshold := cfg.GetInt(flags.FlagRiskScoreThreshold)
+	riskScoreTest := riskScoreFFsEnabled || riskScoreThreshold != -1
+
+	reachability := cfg.GetBool(flags.FlagReachability)
```

Comment:
We have the same logic in `internal/commands/ostest/workflow.go` (line 380 onwards), where it is used for logging as far as I can see.

Do you think we could replace the logic in workflow.go and use the flow we return for logging purposes.